modules/cluster-logging-collector-log-forward-es.adoc

```diff
@@ -0,0 +1,104 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-external.adoc
+
+[id="cluster-logging-collector-log-forward-es_{context}"]
+= Forwarding logs to an external Elasticsearch instance
+
+By default, the Cluster Logging Operator forwards all logs to the internal {product-title} Elasticsearch instance. However, you can optionally forward logs to an external Elasticsearch v5.x or v6.x instance in addition to or instead of the internal instance. You are responsible for configuring the external log aggregator to receive the logs from {product-title}.
```

Comment:
Start from 4.5, we use ES v6.x, so I think maybe we should say `you can optionally forward logs to an external Elasticsearch v6.x instance`.

Reply:
@sichvoge What did we decide on this?